### ISSUE TITLE
Some rules in Essential Eight (e8) profile are missing CCE/OVAL/OCIL (RHEL8)

### ISSUE THREAD

**Issue by matusmarhefka** [ComplianceAsCode/content#4912]
#### Description of problem:
Stats for the Australian Cyber Security Centre (ACSC) Essential Eight (e8) profile in RHEL8:

The following rules are missing CCE:
```
package_talk-server_removed
package_ypbind_removed
package_telnet-server_removed
package_rsh_removed
package_rsh-server_removed
package_quagga_removed
service_avahi-daemon_disabled
package_squid_removed
service_squid_disabled
rpm_verify_ownership
network_sniffer_disabled
sudo_remove_nopasswd
sudo_remove_no_authenticate
sudo_require_authentication
audit_rules_execution_setfiles
sshd_print_last_log
sshd_set_loglevel_info

```

The following rules are missing OVAL:
```
file_permissions_unauthorized_suid
```

The following rules are missing OCIL:
```
security_patches_up_to_date
mount_option_dev_shm_nodev
mount_option_dev_shm_nosuid
mount_option_dev_shm_noexec

```

#### SCAP Security Guide Version:
upstream master

**Comment by matejak** [comment]
Missing OCIL has been fixed by #4910, OCIL for `security_patches_up_to_date` is not needed (see [comment](https://github.com/ComplianceAsCode/content/issues/4913#issuecomment-540450954) in #4913).

**Comment by matusmarhefka** [comment]
All issues have been solved. Thanks @matejak
